tests/unit/samples/wiringfastapi/web.py

```diff
@@ -1,7 +1,6 @@
 import sys
 
 from fastapi import FastAPI, Depends
-from fastapi import Request  # See: https://github.com/ets-labs/python-dependency-injector/issues/398
```

Comment:
Could you, please, leave this import? There was a bug when importing ``Request`` crashed wiring. Follow the link for the details.

Reply:
reverted the change :sparkles: